sixtyfps_runtime/corelib/model.rs

```diff
@@ -231,23 +267,16 @@ impl<T> core::cmp::PartialEq for ModelHandle<T> {
     fn eq(&self, other: &Self) -> bool {
         match (&self.0, &other.0) {
             (None, None) => true,
-            (None, Some(_)) => false,
-            (Some(_), None) => false,
-            (Some(a), Some(b)) => Rc::ptr_eq(a, b),
+            (Some(a), Some(b)) => a == b,
+            _ => false,
         }
     }
 }
 
-impl<T> From<Rc<dyn Model<Data = T>>> for ModelHandle<T> {
```

Comment:
Just curious, why was this removed?

Reply:
I did not need it to build, but then I apparently did not build everything somehow.

At least it did build and test just fine on my local machine and fails to build here.

Reply:
IMO is worth keeping this public API unless there’s a good reason to remove it (such as clippy complaining). IIRC initially the Handle had just the Rc but we had the add the Option, and then this impl was added to make the API easier to use.

Reply:
I will add this back in in the next push.